ci-operator/config/jboss-fuse/camel-quarkus-openshift-interop/jboss-fuse-camel-quarkus-openshift-interop-main__camel-quarkus-ocp4.16-lp-interop.yaml

```diff
@@ -41,11 +43,9 @@ tests:
       FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
```

Comment:
Doesn't this need to change?

Reply:
Thanks for catching that, I have updated it now